Source/Windows10/Prism.Windows/PrismApplicationBase.core.cs

```diff
@@ -207,16 +216,13 @@ protected void InitializeModules()
             }
         }
 
-        public abstract IContainerExtension CreateContainerExtension();
+        protected abstract IContainerExtension CreateContainerExtension();
 
         protected virtual void RegisterRequiredTypes(IContainerRegistry containerRegistry)
         {
-            // don't forget there is no logger yet
-            Debug.WriteLine($"{nameof(PrismApplicationBase)}.{nameof(RegisterRequiredTypes)}()");
-
-            // required for view-models
-
-            containerRegistry.Register<INavigationService, NavigationService>(NavigationServiceParameterName);
+            containerRegistry.Register<IPlatformNavigationService, NavigationService>(NavigationServiceParameterName);
```

Comment:
why do we have a secondary registration for IPLatformNavigationService? This should not be needed.